README.md

```diff
@@ -62,85 +62,39 @@ Then start the `xremap` daemon by running:
 sudo xremap config.yml
 ```
 
-(You will need to leave it running for your mappings to take effect.)
+To use it without sudo (recommended) run the following commands:
 
-<details>
-<summary>If you want to run xremap without sudo, click here.</summary>
-
-### Running xremap without sudo
-
-To do so, your normal user should be able to use `evdev` and `uinput` without sudo.
-In Ubuntu, this can be configured by running the following commands and rebooting your machine.
-
-```bash
+```sh
 sudo gpasswd -a YOUR_USER input
-echo 'KERNEL=="uinput", GROUP="input", TAG+="uaccess"' | sudo tee /etc/udev/rules.d/input.rules
-```
-
-#### Arch Linux
-
-The following can be used on Arch.
-
-```bash
-lsmod | grep uinput
-```
-If this module is not loaded, add to `/etc/modules-load.d/uinput.conf`:
-```bash
-uinput
-```
-Then add udev rule.
-
-```bash
-echo 'KERNEL=="uinput", GROUP="input", TAG+="uaccess"' | sudo tee /etc/udev/rules.d/99-input.rules
+echo 'KERNEL=="uinput", GROUP="input", TAG+="uaccess"' | sudo tee /etc/udev/rules.d/90-input.rules
+echo 'uinput' | sudo tee /etc/modules-load.d/uinput.conf
```

Comment:
My environment doesn't need this. This should go to platform-specific sections.

Reply:
I'm not sure about this: I just tried a fresh Ubuntu Desktop 22.04 install on a VM and indeed without this setup xremap produces a `Permission denied` error.
Are you sure this is not needed for Ubuntu?

Reply:
Here's my Ubuntu 22.04 environment right now:

```
$ ls -la /etc/modules-load.d/
total 20
drwxr-xr-x   2 root root  4096 Jan 28 19:53 .
drwxr-xr-x 147 root root 12288 Mar 21 08:54 ..
-rw-r--r--   1 root root   119 Apr 11  2022 cups-filters.conf
lrwxrwxrwx   1 root root    10 Nov 21 12:57 modules.conf -> ../modules

$ cat /etc/modules-load.d/cups-filters.conf
# Parallel printer driver modules loading for cups
# LOAD_LP_MODULE was 'yes' in /etc/default/cups
lp
ppdev
parport_pc

$ cat /etc/modules-load.d/modules.conf
# /etc/modules: kernel modules to load at boot time.
#
# This file contains the names of kernel modules that should be loaded
# at boot time, one per line. Lines beginning with "#" are ignored.
```

As I've set up Ubuntu environments from scratch with these instructions multiple times myself, I'm pretty sure it has not been necessary at least for Ubuntu 20.04 and 22.04.

Reply:
In other words, Ubuntu loads uinput by default. This is not the case for highly customizable distributions like Arch Linux, but it sounds very normal that distributions like Ubuntu have this kind of thing out of the box.

Reply:
Understood, hopefully my latest commit makes this more distro-specific